index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>10-Gano Courses</title>
+</head>
+<body>
+  <nav class="black-navbar">
+    <ul class="social-icons-container">
+      <li><img src="assets/Rectangle.png" alt="fb-icon"></li>
+      <li><img src="assets/twitter.png" alt="twitter-icon"></li>
+      <li><a>English</a></li>
+      <li><a>My Page</a></li>
+      <li><a>Logout</a></li>
+    </ul>
+  </nav>
+  <nav class="navbar" id="navbar">
+    <div class="logo-container">
+      <a href="index.html">
+        <img src="assets/10-gano-high-resolution-logo-black-on-transparent-background.png" alt="cc-logo" />
+      </a>
+    </div>
+    <div class="bars" id="toggle-icon" onclick="toggleMobileMenu(this)">
+      <div class="bar1"></div>
+      <div class="bar2"></div>
+      <div class="bar3"></div>
+      <ul class="nav-menu">
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a>Program</a>
+        </li>
+        <li>
+          <a>Join</a>
+        </li>
+        <li>
+          <a>Sponsor</a>
+        </li>
+        <li>
+          <a>News</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="links-container">
+    <ul>
+      <li>
+        <a href="about.html">About</a>
+      </li>
+      <li>
+        <a>Program</a>
+      </li>
+      <li>
+        <a>Join</a>
+      </li>
+      <li>
+        <a>Sponsor</a>
+      </li>
+      <li>
+        <a>News</a>
+      </li>
+    </ul>
+    <button class="cc-button" type="button">CC Campaign</button>
```

Comment:
- [ ] Great job implementing the request by the previous reviewer. However, your design does not still match the guiding template. Kindly align the dark top part of your navbar with the white bottom part of your navbar by the right as requested by the previous reviewer. Look at the both designs and work your design alongside the guiding design.

### Your Design
![your-nav-align](https://user-images.githubusercontent.com/26597432/225891254-3a7a04bd-7b3b-43c3-a3a0-8d69ea7421c4.png)

### Guiding Design
![temp-nav-align](https://user-images.githubusercontent.com/26597432/225891302-452ec25b-0361-4092-89db-fa64f7925f8d.png)